<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PkgDetailsWidget</class>
 <widget class="QWidget" name="PkgDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="IndicatorWidget" name="pkgDetailsIndicator" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="pkgDetailsStack">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="pkgDlSizeLabel_2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pkgSpaceReqLabel_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="pkgListsTab">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pkgUpgradeTab">
          <attribute name="title">
           <string>Packages to upgrade</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QTreeWidget" name="pkgUpgradeList">
             <column>
              <property name="text">
               <string>Package name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Installed version</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New version</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_9">
          <attribute name="title">
           <string>Packages to install</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QTreeWidget" name="pkgInstallList">
             <column>
              <property name="text">
               <string>Package name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Version</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_10">
          <attribute name="title">
           <string>Packages to reinstall</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QTreeWidget" name="pkgReinstallList">
             <column>
              <property name="text">
               <string>Package name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Version</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Reason</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Update log:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="pkgUpdateLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IndicatorWidget</class>
   <extends>QWidget</extends>
   <header location="global">indicatorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
